va/android/va_android.cpp

```diff
@@ -28,14 +28,12 @@
 #include "va_backend.h"
 #include "va_internal.h"
 #include "va_trace.h"
-#include "va_fool.h"
 #include "va_android.h"
 #include "va_drmcommon.h"
 #include "va_drm_utils.h"
 #include <stdarg.h>
 #include <unistd.h>
 #include <sys/types.h>
-#include <sys/stat.h>
```

Comment:
Drop this include is the only thing which bother me a little. For example, https://manpages.ubuntu.com/manpages/xenial/man2/open.2.html:
```
       #include <sys/types.h>
       #include <sys/stat.h>
       #include <fcntl.h>
```

Though there are other mans over network which just list `#include <fcntl.h>`. Maybe safer to leave this include here? I am not sure how android includes would be structured.

Reply:
AFAICT there is lots of magic about these includes - both C runtime (glibc vs bionic) and version specific. So it's safer to keep it as-is. Pushed an update with the include preserved.